ui/ecsconfig.py

```diff
@@ -480,25 +480,33 @@ def add_all():
         return None
 
     if l:
```

Comment:
It's not a change you made in this PR, but I find it difficult to remember what these single-letter vars mean. I think it'd be a good idea to rename them to something more meaningful (in a different PR, of course).